src/main/java/frc/robot/commands/Inake.java

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.subWristem;
+
+public class Inake extends CommandBase {
+  subWristem intake = new subWristem();
+  public Inake() {
+    // Use addRequirements() here to declare subsystem dependencies.
```

Comment:
pass the intake as an argument in the constructor and use the addRequirements method with the intake.